app/controllers/thesis_controller.rb

```diff
@@ -86,6 +94,24 @@ def process_theses_update
 
   private
 
+  def deleted_file_list
+    list = []
+    return list if not thesis_params["files_attachments_attributes"]
+    thesis_params["files_attachments_attributes"].values.select {
```

Comment:
Hmmm. I'm not sure what to do with this. We generally prefer `do...end` for multiline blocks but also prefer `{}` for chaining.

I think the line at the end of this style guide rule might be in play here:
```
is this code really readable and can the blocks' contents be extracted into nifty methods?

```

Reply:
Adding the link might help:
https://github.com/rubocop/ruby-style-guide#single-line-blocks

Reply:
Thanks for sharing this link, I probably need to look over the entire style guide a bit more.

I tried reworking this method, looking for ways to improve legibility - can you take a look at the commit that I just pushed up? I'm also happy to jump into a pairing session on this if talking live would help.